tendrl/commons/flows/__init__.py

```diff
@@ -115,7 +115,10 @@ def _execute_atom(self, atom_fqdn):
             ns, atom_name = atom_fqdn.split(".atoms.")
             ns, obj_name = ns.split(".objects.")
 
-            runnable_atom = self._ns.get_atom(obj_name, atom_name)
+            if hasattr(self, "obj"):
+                runnable_atom = self.obj._ns.get_atom(obj_name, atom_name)
+            else:
+                runnable_atom = self._ns.get_atom(obj_name, atom_name)
```

Comment:
In case of object specific atoms like `object.Pool.atoms.delete.Delete, it was showing error `"self._ns" ` not defined. By debugging I found that `_ns` is defined under `self.obj` and so the changes.
Without this I was not able to test delete/resize/update pools and rbds.